cmake/kokkos_tribits.cmake

```
@@ -79,6 +79,12 @@ MACRO(KOKKOS_PROCESS_SUBPACKAGES)
   if (KOKKOS_HAS_TRILINOS)
     TRIBITS_PROCESS_SUBPACKAGES()
   else()
+  # AJP & DZP added for RPS - based perf testing
+  if (KOKKOS_ENABLE_TRACKED_TESTING)
+    SET(INFRASTRUCTURE_ONLY ON CACHE BOOL "Only build the RAJAPerf infrastructure,no builtin kernels")
```

Why does this variable need to be cached?
```suggestion
    SET(INFRASTRUCTURE_ONLY ON CACHE BOOL "Only build the RAJAPerf infrastructure, no builtin kernels")
```

Also, what does this mean? Are all normal unit tests not run when `RAJAPerf` is enabled to be run?